### PR TITLE
update top-level folder name

### PR DESCRIPTION
**Pull request by ericyz**

### CODE REVIEW

**Comment by rjerrems** [comment]
This is to address this issue https://github.com/terraform-google-modules/terraform-example-foundation/issues/12

**Comment by rjerrems** [comment]
I think if this is a generic top level folder, like products - it should probably move to the 1-org step. I would however argue that we should avoid adding something at the top level unless it is adding value given the hierarchy only goes 4 layers deep. What I would advocate is a hierarchy like this:

- business unit / product area (folder)
-- product/app (folder)
--- prod & nonprod (folders)
---- product widget service (projects)
---- product foo service (projects)

That way each business unit can have N products/apps and each app can have N services in the prod/non prod folders. It also gives 3 layers which are practical to delegate IAM if a user wishes, whereas a top level folder like `products` or `app` doesn't really provide value other than being a convenient container visually.

What do you think?

**Comment by morgante** [comment]
We should choose one of the hierarchies from go/org-hierarchy-options

File: 3-projects/folders.tf [hunk 1]
```diff
@@ -14,7 +14,19 @@
...
+ *****************************************/
+resource "google_folder" "business_unit" {
+  display_name = "example-business-unit"
+  parent       = "organizations/${var.org_id}"
+}
+
+/******************************************
+  Team Folders
+ *****************************************/
+module "standard_project_folders" {
```
Comment (rjerrems) [inline 3-projects/folders.tf:28]: for consistency, it probably makes sense to call this something like `example_team_folders`

File: 3-projects/folders.tf [hunk 2]
```diff
@@ -14,7 +14,19 @@
  * limitations under the License.
  */
 
-resource "google_folder" "app" {
-  parent       = "organizations/${var.organization_id}"
-  display_name = "app"
+/******************************************
+  Business Unit Folders
+ *****************************************/
+resource "google_folder" "business_unit" {
```
Comment (rjerrems) [inline 3-projects/folders.tf:20]: nit: to help clearly label what this is would be good to have something like `example_business_unit` for resource name

**Pull request merged**